api.planx.uk/modules/webhooks/service/analyzeSessions/operations.ts

```diff
@@ -15,6 +15,7 @@ import type { Operation } from "../sanitiseApplicationData/types.js";
  *  so that Metabase picks up the new columns
  */
 const ALLOW_LIST = [
+  "application.type",
```

Comment:
nit: both `ALLOW_LIST` lists are currently sorted alphabetically, it'd be nice to maintain that order please !